tests/unit/fem/test_pa_kernels.cpp

```diff
@@ -469,7 +470,7 @@ TEST_CASE("PA Convection", "[PartialAssembly]")
 } // test case
 
 // Advanced unit tests for convection
-TEST_CASE("PA Convection advanced", "[PartialAssembly][MFEMData]")
+TEST_CASE("PA Convection advanced", "[PartialAssembly][MFEMData][CUDA]")
```

Comment:
Do we need a `[HIP]` version too?

Reply:
I'm preparing a separate PR with CUDA tests: https://github.com/mfem/mfem/pull/2892